source/common/zettlr-template.js

```diff
@@ -90,7 +90,7 @@ function makeTemplate (cat, tpl, data = {}) {
     let precompiled = require(`./assets/tpl/${cat}/${tpl}.handlebars.js`)
     return handlebars.template(precompiled)(data)
   } catch (e) {
-    global.log.error(e.message, e)
+    console.error(e.message, e)
```

Comment:
My dear, why did you replace the persisting global.log here? :(

(P.S.: Reminds me that I need to splice the stream of logs in the provider and also pipe one part to the STDOUT if the app is running from command line …)

Reply:
It didn't worked. `global.log` was undefined here. Thus I had a look in the files where the make template method was used (I think it was zettlr dialog) and there `console` was used as well. What's the proper way to solve this?

Reply:
Oh wait … yes, I didn't implement the global logging in the renderer process -.-' My bad, just leave console for now! We can deal with that later on :)